src/main/java/com/ciandt/techgallery/service/endpoint/TechnologyEndpoint.java

```diff
@@ -64,15 +68,20 @@ public Response getTechnologies() throws InternalServerErrorException, NotFoundE
   public Response getTechnology(@Named("id") String id) throws NotFoundException {
     return service.getTechnology(id);
   }
-  
+
   @ApiMethod(name = "findByFilter", path = "technology/search", httpMethod = "get")
-  public Response findTechnologyByFilter(User user,
-      @Named("titleContains") String titleContains,
+  public Response findTechnologyByFilter(User user, @Named("titleContains") String titleContains,
       @Named("shortDescriptionContains") String shortDescriptionContains,
-      @Named("recommendationIs") String recommendationIs)
-          throws ServiceException {
-    return service.findTechnologiesByFilter(
-        new TechnologyFilter(titleContains, shortDescriptionContains, recommendationIs), user);
+      @Named("recommendationIs") String recommendationIs,
+      @Named("orderOptionIs") String orderOptionIs) throws ServiceException {
+    return service.findTechnologiesByFilter(new TechnologyFilter(titleContains,
+        shortDescriptionContains, recommendationIs, orderOptionIs), user);
+  }
+
+  @ApiMethod(name = "getOrderOptions", path = "technology-getOrderOptions", httpMethod = "post")
```

Comment:
@felipeaziz please do not use verbs on resources names. HTTP verbs should be used instead.